api/src/logic.ts

```diff
@@ -31,11 +31,27 @@ export function fetchProducts() {
   );
 }
 
+export function fetchPlans(id: String) {
+  return axios
+    .get(`${API}/products/${id}/plans`)
+    .then((result) => {
+      const data = result.data[0][0];
```

Comment:
The main idea is to get all the available plans.
For example for https://iq.dev.inrix.com/api/products/roadway-analytics/plans we receive 4 plans - 2 individual and 2 enterprise. The goal is to get all of these visualized in the grid.